### ISSUE TITLE
What is the computational cost of training?

### ISSUE THREAD

**Issue by lovecambi** [X-PLUG/mPLUG-Owl#29]
How many GPUs (V100 or A100) are required?

**Comment by MAGAer13** [comment]
For the first stage training it requires 32 A100 GPUs. For the second instructional tuning, a single A100 is sufficient.

**Comment by MAGAer13** [comment]
We provide both first and second stage checkpoints for convience.

**Comment by JianHu-1** [comment]
> For the first stage training it requires 32 A100 GPUs. For the second instructional tuning, a single A100 is sufficient.

 A100 40GB? or  80GB?

**Comment by MAGAer13** [comment]
40G is sufficient. But 80Gb is better. See #35

**Comment by lovecambi** [comment]
> 40G is sufficient. But 80Gb is better. See #35

In the paper, batch size = 256 for instruction tuning, in this setup, how many GPU is required?

**Comment by MAGAer13** [comment]
You can use the gradient accumulation technique to enlarge your batch size without increasing the number of GPUs.
